### PR TITLE
Ubuntu version 20.04

### LINKED ISSUE #324: Release first layer of docker images
After updating each docker image one by one we are ready to start drafting releases for each of them, for the beginning let's merge `276-ubuntu-version-20` changes with `master` and draft releases for first layer of images

### PR DESCRIPTION
**Pull request by ArDrambyan**
Closes #324 

## Changes

Resolved conflicts with the master branch

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
             echo "$DOCKERHUB_PASSWORD" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
 
             echo List of updated files
-            FILES=$(git diff-tree --name-only -r --diff-filter=ACMRT origin/master HEAD)
+            FILES=$(git diff-tree --name-only -r --diff-filter=ACMRT origin/276-ubuntu-version-20 HEAD)
```
Comment (menewman) [inline .circleci/config.yml:20]: Should we change this back to `origin/master` before merging?

**Review by menewman — APPROVED** [review]
Phew, what a big merge! Nice to see it all coming together. Approving this overall since the images should all be working individually, but with one small comment about the CircleCI/Git change detection.

File: .circleci/config.yml (hunk 1 above)
Comment (ArDrambyan) [reply .circleci/config.yml:20]: yeah, I forgot to revert it, thanks!

**Comment by ArDrambyan** [comment]
@menewman - CI is failing because it tries to rebuild everything :D

**Comment by menewman** [comment]
@ArDrambyan - If it's just a one time failure due to the way we calculate changed files in CircleCI, I think the answer is probably a GitHub admin override, which I'm happy to help with. But also, the other thing we should do is release these base updates in layers? First, base. Then, update the Dockerfiles to all inherit from base v9.0, then the next layer of dependencies?

**Comment by ArDrambyan** [comment]
> If it's just a one time failure due to the way we calculate changed files in CircleCI, I think the answer is probably a GitHub admin override, which I'm happy to help with

It would be great!





> But also, the other thing we should do is release these base updates in layers? First, base. Then, update the Dockerfiles to all inherit from base v9.0, then the next layer of dependencies?

I think yes, maybe I'll create intermediate branches to test dockerfiles chunk by chunk but the main idea is to do deploy them layers by layers

**Review by menewman — CHANGES_REQUESTED** [review]
Based on our conversation in Slack, I'm setting status to "request changes" to await the strategy of splitting into small PRs that we can merge and release, layer by layer

**Pull request closed without merging**